slam_toolbox/lib/karto_sdk/src/Mapper.cpp

```diff
@@ -2758,8 +2758,11 @@ namespace karto
 			  m_pSequentialScanMatcher->MatchScan(pScan,
 					  m_pMapperSensorManager->GetRunningScans(pScan->GetSensorName()),
 					  bestPose,
-					  covariance);
+					  cov);
 			  pScan->SetSensorPose(bestPose);
+			  if (covariance) {
+          *covariance = cov;
```

Comment:
Ditto

Reply:
> LGTM beyond formatting issues if you tested working.
> 
> Also, this should be against the `noetic-devel` branch since Melodic is EOL (though there's no reason you can't use the noetic branch on melodic)

Can I cherry-pick it on `noetic-devel` ?